arangodb-net-standard/GraphApi/GraphApiClient.cs

```diff
@@ -512,5 +512,34 @@ public async Task<GetGraphEdgeCollectionsResponse> GetEdgeCollectionsAsync(strin
                 throw await GetApiErrorException(response);
             }
         }
+
+        /// <summary>
+        /// Change one specific edge definition.
+        /// This will modify all occurrences of this definition in all graphs known to your database.
+        /// PUT/_api/gharial/{graph}/edge/{definition}
+        /// </summary>
+        /// <param name="graphName"></param>
+        /// <param name="collectionName"></param>
+        /// <param name="body"></param>
+        /// <param name="query"></param>
+        /// <returns></returns>
+        public async Task<PutGraphDefinitionResponse> PutGraphDefinitionAsync(string graphName, string collectionName, PutGraphDefinitionBody body, PutGraphDefinitionQuery query = null)
```

Comment:
From what I read in the doc, it will "Replace an edge definition". I think a more appropriate name would be `PutEdgeDefinitionAsync`.